planning/behavior_velocity_planner/include/scene_module/occlusion_spot/occlusion_spot_utils.hpp

```diff
@@ -183,13 +183,15 @@ struct PossibleCollisionInfo
 struct DebugData
 {
   double z;
+  double baselink_to_front;
   std::string road_type = "";
   std::string detection_type = "";
   Polygons2d detection_area_polygons;
   std::vector<lanelet::BasicPolygon2d> close_partition;
   std::vector<geometry_msgs::msg::Point> parked_vehicle_point;
   std::vector<PossibleCollisionInfo> possible_collisions;
   std::vector<geometry_msgs::msg::Point> occlusion_points;
+  std::vector<geometry_msgs::msg::Pose> debug_poses;
   void resetData()
   {
     close_partition.clear();
```

Comment:
This change is also necessary to remove previous marker
```suggestion
    debug_poses.clear();
    close_partition.clear();
```

And I tested with this change.
![image](https://user-images.githubusercontent.com/65527974/209971030-034ed459-9144-4261-9a60-9ac565d9c0da.png)